### ISSUE TITLE
Close all rows

### ISSUE THREAD

**Issue by ghmendonca** [jemise111/react-native-swipe-list-view#465]
Hi, I need to close all rows at the same time, but looking the source code, I couldn't find a way to do this.

I think it could be a function called closeAll that lives inside SwipeListView, or a way to access each list item ref to call the closeRow function.

**Comment by jemise111** [comment]
Hi @ghmendonca I provided a way to do this in this issue: https://github.com/jemise111/react-native-swipe-list-view/issues/242

If this doesn't solve your problem please re-open and let me know, thanks!

**Comment by ghmendonca** [comment]
@jemise111 it doesn't work in my case. In my case I'm passing a custom component which renders the SwipeRow, so all the functions and props passed in the list view doesn't work, since these functions are called by the SwipeRow component. 

Here's my code:
```
<SwipeListView<Group>
                data={filteredGroups}
                onRowDidOpen={() => {
                    console.log("this doesn't fire")
                }}
                renderItem={({ item: group }) => (
                    <ListItem group={group} />
                )}
                closeOnRowOpen
                keyExtractor={(group) => group.iId.toString()}
                ItemSeparatorComponent={() => <Separator style={{ backgroundColor: 'rgba(166, 176, 189, 0.5)' }} />}
            />
```

And my ListItem:
```
<SwipeRow
            disableLeftSwipe={!group.boHasSubgroups}
            rightOpenValue={-240}
            disableRightSwipe
            ref={swipeRowRef}
            onRowDidClose={() => {
                setIsOpen(false)
                setRoundedBorder(false)
            }}
            onSwipeValueChange={() => setRoundedBorder(true)}
            onRowDidOpen={() => setIsOpen(true)}
            onRowOpen={() => setRoundedBorder(true)}
        >
            ...
        </SwipeRow>
```

Wouldn't be better if you keep tracking the list items refs inside the list view component? So I could just use the ref of the list view to manage the list items.

**Comment by jemise111** [comment]
@ghmendonca hm well you can still pass the refs from the SwipeRow up to your custom component.

I will try to implement this feature over the weekend, but in the meantime I think this is still achievable

**Comment by ghmendonca** [comment]
Awesome! I'll implement passing the refs up while you don't release this feature.

Thanks man :)

**Comment by jemise111** [comment]
@ghmendonca done! This was fixed by #466 

Grab a ref to the SwipeListView and call `closeAllOpenRows()` on it

LMK if you have questions

![RPReplay_Final1589372580 MP4](https://user-images.githubusercontent.com/4265163/81812945-738c7900-94ec-11ea-8a7a-55f1a591ef20.gif)

**Comment by Prashantsam** [comment]
> @ghmendonca done! This was fixed by #466
> 
> Grab a ref to the SwipeListView and call `closeAllOpenRows()` on it
> 
> LMK if you have questions
> 
> ![RPReplay_Final1589372580 MP4](https://user-images.githubusercontent.com/4265163/81812945-738c7900-94ec-11ea-8a7a-55f1a591ef20.gif)

@jemise111  It would be very helpful if you could share me the above reference code of it.

**Comment by rampark** [comment]
> @ghmendonca done! This was fixed by #466
> 
> Grab a ref to the SwipeListView and call `closeAllOpenRows()` on it
> 
> LMK if you have questions
> 
> ![RPReplay_Final1589372580 MP4](https://user-images.githubusercontent.com/4265163/81812945-738c7900-94ec-11ea-8a7a-55f1a591ef20.gif)

@jemise111 It would be very helpful if you could share me the above reference code of it.
